.github/workflows/test-docker.yml

```
@@ -0,0 +1,40 @@
+name: Test Docker
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        docker: [
+          alpine,
+          arch,
+          ubuntu-16.04-xenial-amd64,
+          ubuntu-18.04-bionic-amd64,
+          debian-9-stretch-x86,
+          debian-10-buster-x86,
+          centos-6-amd64,
+          centos-7-amd64,
+          amazon-1-amd64,
+          amazon-2-amd64,
+          fedora-29-amd64,
+          fedora-30-amd64,
+        ]
+        dockerTag: [master]
+
+    name: ${{ matrix.docker }}
+
+    steps:
+    - uses: actions/checkout@v1
```

You'd rather `@v1` instead of
```suggestion
    - uses: actions/checkout@master
```
?

---

Yep:

> We strongly recommend that you include the version of the action you are using by specifying a Git ref, SHA, or Docker tag number. If you don't specify a version, it could break your workflows or cause unexpected behavior when the action owner publishes an update.
> 
> - Using the commit SHA of a released action version is the safest for stability and security.
> - Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.
> - Using the `master` branch of an action may be convenient, but if someone releases a new major version with a breaking change, your workflow could break.

https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepsuses